src/components/FileUpload/index.js

```diff
@@ -35,14 +35,12 @@ class FileUpload extends React.Component {
   };
 
   render() {
-    const { file, disabled, loading, filesize, filename } = this.props;
+    const { disabled, loading, filesize, filename } = this.props;
     const { filesize: filesizeDefault, filename: filenameDefault } = this.state;
     const disabledState = loading || disabled;
 
-    let uploadStyles = styles.fileUpload;
```

Comment:
We weren't using these styles, but that might be a mistake we want to correct.

Reply:
hmm - that's interesting. cool - you need to remove it in both spots in the file though so it doesn't return an error.

Reply:
ah looks like you did.